Topics/Tech_Stacks/Django_Rest.md

```diff
@@ -46,13 +49,121 @@ Django Rest Framework is a popular library for building RESTful APIs with Django
 └── manage.py
  ```
  
- In order to add Django Rest Framework to your project, open the settings.py file and add 'rest_framework' to the INSTALLED_APPS list: ```INSTALLED_APPS = [..., 'rest_framework']```
+ In order to add Django Rest Framework to your project, open the settings.py file and add 'rest_framework' to the INSTALLED_APPS list: 
+ ```
+INSTALLED_APPS = [
+    ..., 
+    'rest_framework',
+]
+ ```
  
+ While in the settings.py file, you should also add:
+```
+REST_FRAMEWORK = {
```

Comment:
You could add an external link aroudn this section that points towards more configuration options that go in the settings.py file.

Reply:
Thanks for the suggestion, I will add that